vscode/src/commands/register.ts

```diff
@@ -35,7 +36,8 @@ const commandModules: ICommandModules = {
     buildOperations: registerBuildOperationCommands,
     refactor: registerRefactorCommands,
     util: registerUtilCommands,
-    debug: registerDebugCommands
+    debug: registerDebugCommands,
+    runConfiguration: registerRunConfigurationCommands // removed the conditional command -> that doesn't make sense 
```

Comment:
If you are sure about this then no need of comment IMO.

Reply:
okay